.gitignore

```diff
@@ -4,4 +4,5 @@ cypress/downloads/
 cypress/integration/1-getting-started/
```

Comment:
cypress/fixtures/ needs for storing test data, if u have same test data u can replace there and delete this folder from the gitignore